schema.sql

```diff
@@ -9,5 +9,20 @@ CREATE TABLE animals (
 );
 ALTER TABLE Animals
 ADD COLUMN species VARCHAR(255);
+CREATE TABLE owners (
+    id serial PRIMARY KEY,
+    full_name varchar(255),
+    age integer
+);
+
+CREATE TABLE species (
+    id serial PRIMARY KEY,
+    name varchar(255)
+);
+
+ALTER TABLE animals
+    ADD COLUMN species_id integer REFERENCES species(id),
+    ADD COLUMN owner_id integer REFERENCES owners(id);
+
```

Comment:
- [ ] Great job on modifying the animals table by adding species_id however please also remove column `species` from `animals` table as it would be redundant and unnecessary.